internal/cli/main.go

```diff
@@ -122,10 +123,11 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n", createUsage, gotoU
 	// initialize migrate
 	// don't catch migraterErr here and let each command decide
 	// how it wants to handle the error
-	migrater, migraterErr := migrate.New(*sourcePtr, *databasePtr)
+	ctx := context.Background()
```

Comment:
This is the only place where passing the context is clearly an improvement.
Changing every other place without an explicit use case seems excessive 🤷

Reply:
I have an explicit use case: adding OpenTelemetry support to this project so that the progress can be followed in the observability tool of your choice with additional information on what's happening and what exactly went wrong in case of an error.

This relies heavily on passing around context, but I don't want to spend more time on it (outside of my internal WIP) if this particular change isn't viable or desired.

I acknowledge that on its own it's pretty useless and mostly adds noise to the existing code while breaking backward compatibility. 🤷